File: main-svc/src/test/java/com/chilly/main_svc/containers/ContainersEnvironment.java
package com.chilly.main_svc.containers;

import org.testcontainers.junit.jupiter.Container;
import org.testcontainers.junit.jupiter.Testcontainers;

@Testcontainers
public class ContainersEnvironment {

    @Container
    public static PostgresTestContainer postgresTestContainer = PostgresTestContainer.getInstance();
}

File: business-users-svc/src/test/kotlin/org/chilly/business/users/containers/ContainersEnvironment.kt
package org.chilly.business.users.containers

import org.chilly.common.test.containers.EurekaTestContainer
import org.chilly.common.test.containers.PostgresTestContainer
import org.testcontainers.junit.jupiter.Container
import org.testcontainers.junit.jupiter.Testcontainers

@Testcontainers
open class ContainersEnvironment {

    companion object {
        @Container
        val postgresTestContainer: PostgresTestContainer = PostgresTestContainer.getInstance()
        @Container
        val eurekaTestContainer: EurekaTestContainer = EurekaTestContainer.getInstance()
    }
}
